SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift

```diff
@@ -237,10 +273,17 @@ extension SKYChatConversationViewController {
             msgSenderName = senderName
         }
 
+        if msg.attachment == nil {
```

Comment:
Here we should check the `messageMediaData`, not `attachment`. It is because the message can have attachment that is not a media data, say a `doc` files. In that case, the `media` `JSQMessage` get is nil and result in exception.